<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="35793" started-at="2017-11-01T09:51:45Z" finished-at="2017-11-01T09:52:20Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="35793" started-at="2017-11-01T09:51:45Z" finished-at="2017-11-01T09:52:20Z">
      <class name="fristproject1.LoginTestCase">
        <test-method status="PASS" signature="openbrowser()[pri:0, instance:fristproject1.LoginTestCase@5afa04c]" name="openbrowser" is-config="true" duration-ms="35688" started-at="2017-11-01T15:21:45Z" finished-at="2017-11-01T15:22:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openbrowser -->
        <test-method status="SKIP" signature="BlankErrorTest()[pri:1, instance:fristproject1.LoginTestCase@5afa04c]" name="BlankErrorTest" duration-ms="0" started-at="2017-11-01T15:22:20Z" data-provider="Testdp" finished-at="2017-11-01T15:22:20Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void fristproject1.LoginTestCase.BlankErrorTest() requires a @DataProvider named : Testdp]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void fristproject1.LoginTestCase.BlankErrorTest() requires a @DataProvider named : Testdp
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:263)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:419)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BlankErrorTest -->
        <test-method status="SKIP" signature="BValidTest()[pri:2, instance:fristproject1.LoginTestCase@5afa04c]" name="BValidTest" duration-ms="0" started-at="2017-11-01T15:22:20Z" data-provider="Testdp" finished-at="2017-11-01T15:22:20Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void fristproject1.LoginTestCase.BValidTest() requires a @DataProvider named : Testdp]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void fristproject1.LoginTestCase.BValidTest() requires a @DataProvider named : Testdp
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:263)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:419)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BValidTest -->
      </class> <!-- fristproject1.LoginTestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
